src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -259,4 +294,14 @@ public function equals(self $option): bool
             && $option->isValueOptional() === $this->isValueOptional()
         ;
     }
+
+    /**
+     * Returns true if the option allows $mode.
+     *
+     * @return bool true if mode is $mode, false otherwise
+     */
+    protected function hastMode(int $mode): bool
```

Comment:
Is this different from 'hasMode' ? I may not know the term

Reply:
I just missed the t in the change from `getMode` to `hasMode`

Reply:
I really did not know 😅